### PR TITLE
Project Documentation 1

### LINKED ISSUE #10: Upload project progress 1 to repo

### PR DESCRIPTION
**Pull request by NLmeng**
closes #10

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+.env.development.local
+.env.test.local
+.env.production.local
+.env.local
+
+# parcel-bundler cache (https://parceljs.org/)
+.cache
+.parcel-cache
+
+# Next.js build output
```
Comment (Graywing13) [inline .gitignore]: I think some of this is legacy, since we aren't using nextjs anymore?

**Review by Graywing13 — COMMENTED** [review]
Looks good generally, just a small thing

File: .gitignore (hunk 1 above)
Comment (NLmeng) [reply .gitignore]: thats true i just copied some old gitignore

**Review by Graywing13 — APPROVED** [review]
lgtm :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+User input
+
+![User Input](mocks/mock-a.png)
+
+### App Response: Plan Overview
+
+With sidebar closed
+
+![Plan Overview - Closed Sidebar](mocks/mock-b.png)
```
Comment (L0Lmaker) [inline README.md:92]: Pics not displayed, might be a typo with the markdown

**Review by L0Lmaker — CHANGES_REQUESTED** [review]
Small issue

File: README.md (hunk 2 above)
Comment (NLmeng) [reply README.md:92]: there is no typo

Screenshot 2023-05-22 at 7 31 17 PM

Comment (L0Lmaker) [reply README.md:92]: Turns out mobile view doesn't render the pics lmao

**Review by L0Lmaker — APPROVED** [review]
LGTM

**Pull request merged**